skrf/calibration/calibration.py

```diff
@@ -3794,7 +3795,10 @@ def solve_quadratic(v1, v2, inx, x_est):
             else:
                 # solve for a11/b11, a11 and b11 (use redundant reflect measurement, if available)
                 reflect_est_offset = reflect_est*npy.exp(-2*gamma*reflect_offset) # shift estimated reflect
-                a11_b11 = (reflect_meas_S[:,m,0,0] - a12)/(1 - reflect_meas_S[:,m,0,0]*a21_a11) * (1 + reflect_meas_S[:,m,1,1]*b12_b11)/(reflect_meas_S[:,m,1,1] + b21)
+                # a11_b11 = (reflect_meas_S[:,m,0,0] - a12)/(1 - reflect_meas_S[:,m,0,0]*a21_a11) * (1 + reflect_meas_S[:,m,1,1]*b12_b11)/(reflect_meas_S[:,m,1,1] + b21)
```

Comment:
What's this? If the new code is better this can be removed.

Reply:
To be honest, the first code shouldn't have been there, I initially added it while doing some debugging when I first opened the PR and then realized I didn't delete it. Even now, somehow I ended up commenting it instead of deleting it. I will delete it.